recipes/dcmtk/all/conandata.yml

```diff
@@ -2,25 +2,8 @@ sources:
   "3.6.7":
     url: "https://dicom.offis.de/download/dcmtk/dcmtk367/dcmtk-3.6.7.tar.gz"
     sha256: "7c58298e3e8d60232ee6fc8408cfadd14463cc11a3c4ca4c59af5988c7e9710a"
-  "3.6.6":
```

Comment:
Quick question: are these versions being removed because they are incompatible with the proposed changes and they will not work (and it will be extra work to do it), or just to drop support and support only latest patch?

Reply:
They're dropped because they don't work with the proposed patches - the CMake system of dcmtk is pretty ancient in general unfortunately and uses all sort of interesting CMake magic instead of just using targets which makes everything more complicated.

Reply:
Ok thanks for the explanation. However I see that 3.6.6 is from 2021, even if the CMake is not great, the version itself is relatively new one. I think it is fine, but I prefer to check this removal of versions with the team, and align on what is the best way to move it forward.